src/components/inputs/FormattedNumberInput.tsx

```diff
@@ -54,63 +56,65 @@ export default function FormattedNumberInput({
   }
 
   return (
-    <div className="formatted-number-input">
-      <Form.Item {...formItemProps}>
+    <Form.Item
```

Comment:
**question** are you confident this doesn't break any other uses across the site? 😉 

(i havent checked)

Reply:
Yes had a look around, couldn't spot anything. Don't see how it this change could affect anything user-facing.

Reply:
Awesome sounds good.

Removing a wrapping div could definitely break stuff but lgtm.